nemo/collections/vlm/qwen2vl/data/conversation.py

```diff
@@ -0,0 +1,290 @@
+                        images.append(img)
+        return images
+
+    def to_gradio_chatbot(self):
+        # pylint: disable=C0115,C0116
+        ret = []
+        for i, (role, msg) in enumerate(self.messages[self.offset :]):
+            if i % 2 == 0:
+                if type(msg) is tuple:
+                    msg, image, image_process_mode = msg
+                    if type(image) != list:
+                        image = [image]
+                    if len(image) == 1:
+                        msg = "<image>\n" + msg.replace("<image>", "").strip()
+                    else:
+                        msg = re.sub(r"(<image>)\n(?=<image>)", r"\1 ", msg)
+                    for img in image:
+                        img_b64_str = self.process_image(img, "Default", return_pil=False, image_format="JPEG")
+                        img_str = f'<img src="data:image/jpeg;base64,{img_b64_str}"/>'
+                        msg = msg.replace("<image>", img_str, 1).strip()
+                    if len(msg) > 0:
+                        ret.append([msg, None])
+                else:
+                    ret.append([msg, None])
+            else:
+                ret[-1][-1] = msg
+        return ret
+
+    def copy(self):
+        # pylint: disable=C0115,C0116
+        return Conversation(
+            system=self.system,
+            roles=self.roles,
+            messages=[[x, y] for x, y in self.messages],
+            offset=self.offset,
+            sep_style=self.sep_style,
+            sep=self.sep,
+            sep2=self.sep2,
+            version=self.version,
+        )
+
+    def dict(self):
+        # pylint: disable=C0115,C0116
+        if len(self.get_images()) > 0:
+            return {
+                "system": self.system,
+                "roles": self.roles,
+                "messages": [[x, y[0] if type(y) is tuple else y] for x, y in self.messages],
+                "offset": self.offset,
+                "sep": self.sep,
+                "sep2": self.sep2,
+            }
+        return {
+            "system": self.system,
+            "roles": self.roles,
+            "messages": self.messages,
+            "offset": self.offset,
+            "sep": self.sep,
+            "sep2": self.sep2,
+        }
+
+
+conv_qwen2vl = Conversation(
+    system="",
+    roles=("user", "assistant"),
+    version="qwen2vl",
+    messages=[],
+    offset=0,
+    sep_style=SeparatorStyle.QWEN2VL,
+    sep="",
+)
+conv_chatml_direct = Conversation(
+    system="""<|im_start|>system
+Answer the questions.""",
+    roles=("<|im_start|>user\n", "<|im_start|>assistant\n"),
+    version="",
+    messages=[],
+    offset=0,
+    sep_style=SeparatorStyle.CHATML,
+    sep="<|im_end|>",
+)
+
+default_conversation = conv_qwen2vl
+conv_templates = {
+    "default": conv_qwen2vl,
+    "qwen2vl": conv_qwen2vl,
+    "chatml_direct": conv_chatml_direct,
+}
+
+if __name__ == "__main__":
+    print(default_conversation.get_prompt())
```

Comment:
Can you please search the page. For `print` and `logger` plz change to 
`from nemo.utils import logging` and `logging.info`

for not used print, please remove